README.md

```diff
@@ -1,50 +1,95 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/external-provisioner.svg?branch=master)](https://travis-ci.org/kubernetes-csi/external-provisioner)
 
-# Kubernetes external provisioner that works with CSI volumes
+# CSI provisioner
 
-This is an example external provisioner for Kubernetes which provisions using CSI Volume drivers..  It's under heavy development, so at this time README.md is notes for the developers coding.  Once complete this will change to something user friendly.
+The external-provisioner is a sidecar container that dynamically provisions volumes by calling `ControllerCreateVolume` and `ControllerDeleteVolume` functions of CSI drivers. It is necessary because internal persistent volume controller running in Kubernetes controller-manager does not have any direct interfaces to CSI drivers.
 
-# Build
+## Overview
+The external-provisioner is an external controller that monitors `PersistentVolumeClaim` objects created by user and creates/deletes volumes for them. Full design can be found at Kubernetes proposal at [container-storage-interface.md](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md)
 
-```bash
-make csi-provisioner
-```
+## Compatibility
 
-# Test
+This information reflects the head of this branch.
 
-## Start Kubernetes
+| Compatible with CSI Version                                                                | Container Image                | Min K8s Version |
+| ------------------------------------------------------------------------------------------ | -------------------------------| --------------- |
+| [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-provisioner | 1.14            |
 
-Run a local kubernetes cluster built from latest master code
+## Feature status
 
-## Run Storage Provider
+Various external-provisioner releases come with different alpha / beta features. Check `--help` output for alpha/beta features in each release.
 
-### Use HostPath drivers
+Following table reflects the head of this branch.
 
-Go to drivers and run:
+| Feature    | Status  | Default | Description                                                                                   |
+| ---------- | ------- | ------- | --------------------------------------------------------------------------------------------- |
+| Topology   | Beta    | On      | [Topology aware dynamic provisioning](https://kubernetes-csi.github.io/docs/topology.html)  (requires kubelet 1.14 on nodes). |
```

Comment:
Sorry for flipping back on forth. My latest PR has topology off by default so that the sidecar is backwards compatible with 1.13.

Reply:
Topology is Off by default now.